.github/actions/gradle-command-self-hosted-action/action.yml

```diff
@@ -38,14 +38,9 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Remove default github maven configuration
-      # This step is a workaround to avoid a decryption issue of Beam's
-      # gradle-command-action plugin and github's provided maven
-      # settings.xml file
-      shell: bash
-      run: rm ~/.m2/settings.xml
     - name: Run Gradle Command
       shell: bash
       run: |
+        rm ~/.m2/settings.xml
```

Comment:
```suggestion
      # Removing settings.xml is a workaround to avoid a decryption issue
      # of Beam's gradle-command-action plugin and github's provided
      # maven settings.xml file
        rm ~/.m2/settings.xml
```